packages/util-cli-reporter/lib/js/reporter.js

```diff
@@ -65,7 +65,7 @@ function configureOutput(type, description, message, comment) {
  * @param {String} message the main message
  * @param {String} comment additional comment (optional)
```

Comment:
Use square brackets to make something optional in JSDoc
```suggestion
 * @param {String} [comment=null] additional comment (optional)
```

The same applies to the other methods in here.

Reply:
oh yes, nice. forgot about that